django/contrib/admin/checks.py

```diff
@@ -439,6 +440,19 @@ def _check_field_spec_item(self, obj, field_name, label):
             except FieldDoesNotExist:
                 # If we can't find a field on the model that matches, it could
                 # be an extra field on the form.
+                try:
+                    field = getattr(obj.model, field_name)
+                except AttributeError:
+                    return []
+                if isinstance(field, related.ReverseManyToOneDescriptor):
```

Comment:
I'm not entirely sure this change belongs in this branch, since for `fields` there are currently no checks for M2M either. And the referenced ticket is only about the `list_display`.

Would you agree to separate this check into its own branch with a new ticket?

Reply:
Thanks for the comment, I will open a new ticket for that